requirements.txt

```diff
@@ -1,8 +1,8 @@
 openwisp-controller~=0.7.post1
+openwisp-utils[rest]
```

Comment:
Always pin dependencies, I could do it myself but I want you to do it so you won't forget next time, thanks.

Reply:
@nemesisdesign, you explicitly mentioned to leave it unpinned above https://github.com/openwisp/openwisp-monitoring/pull/228#discussion_r481320603 and your reason for doing so seemed good enough to me.

Reply:
LOL! you're right @nepython, but I forgot already. Let's add a comment to avoid forgetting, can you please add one?